components/multimedia/gst-plugins-good1/Makefile

```diff
@@ -89,6 +103,7 @@ REQUIRED_PACKAGES += library/aalib
 REQUIRED_PACKAGES += library/audio/gstreamer1
```

Comment:
REQUIRED_PACKAGES += audio/wavpack

audio/wavpack is installed on our build server